src/sentry/utils/committers.py

```diff
@@ -128,23 +154,31 @@ def _get_committers(annotated_frames, commits):
                 break
 
     # organize them by this heuristic (first frame is worth 5 points, second is worth 4, etc.)
-    sorted_committers = sorted(committers, key=committers.get)
+    # sorted_committers = sorted(committers, key=Mapping.get)
+    sorted_committers = sorted(committers, key=lambda x: x)
```

Comment:
This might need a closer look and some guidance. 

Original code was `sorted_committers = sorted(committers, key=committers.get)`

The intention is to sort the map by key?

Reply:
I'm not sure sorting by commit author._id is what we want. Given the comments above it sounds like the intent was to sort by score which would have been the values of the map.

Reply:
I'm pretty sure both the comment and the original code want to sort by dict value. If you want to sort by key you don't need to provide a `key=` callback at all.